common/djangoapps/enrollment/data.py

```diff
@@ -40,7 +40,10 @@ def get_course_enrollments(user_id):
         is_active=True
     ).order_by('created')
 
-    enrollments = CourseEnrollmentSerializer(qset, many=True).data
+    if support_user:
+        enrollments = SupportCourseEnrollmentSerializer(qset, many=True).data
```

Comment:
I think instead of making two separate serializers we could handle this condition in `CourseEnrollmentSerializer`.